Utilize the meta theme-color property to make the site look better on…

ismyhc opened the pull request:
This was a small change so that the site would look better on mobile safari, safari compact mode, etc.
<img width="976" alt="Screen Shot 2022-04-27 at 12 44 16 PM" src="https://user-images.githubusercontent.com/840551/165584777-e26eb61e-730c-4112-abbb-0d37f541331a.png">

ismyhc committed:
Utilize the meta theme-color property to make the site look better on mobile safari and other supported browsers

jnordberg reviewed (changes requested):
Code looks good but there is a stray yarn-error.log in there

ismyhc committed:
delete yarn-error.log add to gitignore

ismyhc commented:
> Code looks good but there is a stray yarn-error.log in there

Weird. Yeah, I simply cloned the repo, installed and ran. At any rate, I removed it and also added to .gitignore.
